.github/workflows/autobuild.yml

```diff
@@ -32,11 +32,12 @@ jobs:
       uses: actions/checkout@v1
     - name: Compile
       run: make -j$(sysctl -n hw.logicalcpu)
+      run: ./mac-bundle.sh
```

Comment:
This is actually allowed, though I agree it looks a bit weird. I've changed it to
```yml
run: |
  make -j$(sysctl -n hw.logicalcpu)
  ./mac-bundle.sh
```
Using && isn't necessary because github workflows sets `errexit` on the shell by default, so it already exits on failed commands.